Explorer/Assets/Scripts/ECS/UnityComponents/Systems/UpdateTransformSystem.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Arch.Core;
+using Arch.Core.Extensions;
+using Arch.SystemGroups;
+using Arch.SystemGroups.DefaultSystemGroups;
+using CrdtEcsBridge.Components.Transform;
+using ECS.Abstract;
+using UnityEngine;
+
+[UpdateInGroup(typeof(SimulationSystemGroup))]
+public partial class UpdateTransformSystem : BaseUnityLoopSystem
+{
+    private readonly QueryDescription queryDescription = new QueryDescription().WithAll<SDKTransform, Transform>();
+    private CleanTransform cleanTransform;
```

Comment:
This may be just me but the wording of clean implies resetting to me, but this actually updates the unity transforms.

Reply:
Up! Change the wording please to reflect the actual change of the transform